sexxi_bot.py

```diff
@@ -109,20 +119,21 @@ def run_bot(user_message, start):
     bot.user_input = user_message
     bot.fix_typos()
     if not bot.help_check():
-        if not bot.greeting_check():
-            if not bot.asked_about_self():
-                if not bot.menu_check():
-                    if not bot.weather_check():
-                        if not bot.create_response():  # If all key phrases fail, we gotta actually make a new sentence
-                            bot.response = responses.UNSURE_RESPONSES[0]  # If no response can be created
+        if not bot.fun_check(): #do this before the greeting because we are looking for specific things
+            if not bot.greeting_check():
+                if not bot.asked_about_self():
+                    if not bot.menu_check():
+                        if not bot.weather_check():
+                            if not bot.create_response():  # If all key phrases fail, we gotta actually make a new sentence
+                                bot.response = responses.UNSURE_RESPONSES[0]  # If no response can be created
```

Comment:
its not your code but is there a more "elegant" way of doing this? :stuck_out_tongue: